src/sql/sqlops.proposed.d.ts

```diff
@@ -325,4 +325,28 @@ declare module 'sqlops' {
 			}
 		}
 	}
+
+	/**
+	 * Namespace for interacting with query editor
+	*/
+	export namespace queryEditor {
```

Comment:
Elsewhere we're using all lowercase names for namespaces, e.g. 'objectexplorer'. Not sure if it's right but we should be consistent